src/CONST.js

```diff
@@ -77,6 +77,7 @@ const CONST = {
         TIMEZONE: 'timeZone',
     },
     DEFAULT_TIME_ZONE: {automatic: true, selected: 'America/Los_Angeles'},
+    DEFAULT_ACCOUNT_DATA: {error: '', success: '', loading: true},
```

Comment:
Hm, this kind of gives me a thought, and I am fine with it being a NAB, but I think there are other places that are using this same pattern. I'm primarily thinking about these:

https://github.com/Expensify/Expensify.cash/blob/a740e888cc47b7980b37d7c7c8366d2096fca03f/src/Expensify.js#L22-L26

Which also makes me wonder if that some code should be referencing this constant too?

Reply:
I like this thought. Would also love to see a follow up issue or maybe discussion in `#expensify-open-source` to get more ideas on how to standardize API loading and error cases. Not convinced what we are doing now is always clear or easy to follow, but works for now.

Reply:
Updated Expensify.js to use `DEFAULT_ACCOUNT_DATA` for `[ONYXKEYS.ACCOUNT]`